catalogs/catalog-lakehouse-hudi/src/main/java/org/apache/gravitino/catalog/lakehouse/hudi/backend/hms/HudiHMSTable.java

```diff
@@ -44,15 +51,34 @@ protected HudiHMSTable simpleBuild() {
       table.columns = columns;
       table.indexes = indexes;
       table.partitioning = partitioning;
+      table.sortOrders = sortOrders;
       table.distribution = distribution;
       table.properties = properties;
       table.auditInfo = auditInfo;
       return table;
     }
 
     @Override
-    protected HudiTable buildFromTable(Table backendTable) {
-      // todo: convert HMS table to HudiTable
+    protected HudiHMSTable buildFromTable(Table hmsTable) {
+      name = hmsTable.getTableName();
+      comment = hmsTable.getParameters().get(COMMENT);
+      columns = HiveTableConverter.getColumns(hmsTable, HudiColumn.builder());
```

Comment:
Is Hudi's column type exactly the same as Hive table?

Reply:
According to the table in [the Hudi document](https://hudi.apache.org/docs/sql_ddl#supported-types), the data types supported by Hudi are fewer than those of Hive. I assume they are a subset of Hive. So I think there should be no problem with the conversion here.